modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -83,7 +84,7 @@ const applyFormat = function (ed: Editor, name: string, vars?: FormatVars, node?
     }
 
     // Look for matching formats
-    each(formatList, function (format) {
+    each(formatList, function (format: any) {
```

Comment:
I'm imagining the reason is "because typescript wouldn't work otherwise" but why do we need an explicit any here? I would have thought / hoped that it'd infer the type of format based on the type of formatList if you left the any out.

Reply:
Without this, it'll infer as `unknown` because `formatList` is untyped. The types for that are crazy, so wanted to avoid that. I'll change the above to be `formatList: any[]` instead, as that probably makes more sense.

Reply:
Ahh, wait I can't do that because `formatList` could also be a `Record` :facepalm: That's why I originally made this any instead (as mentioned the formatter is horrible for types and we didn't have much time to investigate).

Reply:
Leave it as is then